<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="rec_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 38pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>단골손님 주문</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 15pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;전화번호&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;뒷자리 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dg_number">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>50</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 25pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;주문하시려는 메뉴를 눌러주세요 !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 20pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;가장 많이 주문한 메뉴&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dg_basic">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>1010</y>
     <width>322</width>
     <height>152</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>322</width>
     <height>152</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 15pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>님의 주문내역</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 20pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;최근 주문 내역&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>571</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="dg_most">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_dg_most">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>102</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>162</width>
        <height>102</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>650</y>
     <width>573</width>
     <height>328</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="dg_last_1">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dg_last_1">
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>102</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="dg_last_2">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dg_last_2">
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>102</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="dg_last_3">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dg_last_3">
        <property name="minimumSize">
         <size>
          <width>162</width>
          <height>102</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="plz_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>1366</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>plz_bg</zorder>
  <zorder>rec_label</zorder>
  <zorder>rec_label_2</zorder>
  <zorder>dg_number</zorder>
  <zorder>rec_label_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>rec_label_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>btn_dg_basic</zorder>
  <zorder>rec_label_6</zorder>
  <zorder>rec_label_7</zorder>
 </widget>
 <resources>
  <include location="mainbg.qrc"/>
 </resources>
 <connections/>
</ui>
